requirements.json

```diff
@@ -55,11 +60,32 @@
     "modulePath": "model/devstudio",
     "bundle": "yes",
     "targetFolderName": "developer-studio",
+    "installAfter": "jdk",
     "dmUrl": "https://devstudio.redhat.com/11/snapshots/builds/devstudio.product_master/latest/all/",
     "url": "https://devstudio.redhat.com/11/snapshots/builds/devstudio.product_master/latest/all/",
-    "filename": "devstudio-11.0.0.AM1-installer-standalone.jar",
+    "fileName": "devstudio-11.0.0.AM1-installer-standalone.jar",
     "sha256sum": "",
-    "version": "11.0.0.AM1"
+    "version": "11.0.0.AM1",
+    "useDownload": true,
+    "requires": "jdk"
+  },
+  "fusetools": {
+    "name": "Red Hat JBoss Fuse Tools",
+    "description": "A certified Eclipse-based integrated development environment (IDE) for Fuse",
+    "vendor": "Red Hat, Inc.",
+    "modulePath": "model/jbossfuse",
+    "bundle": "no",
+    "targetFolderName": "fuse-eap-installer",
+    "installAfter": "devstudio",
```

Comment:
Should be "installAfter": "devstudio" in case install on jbosseap 7.0.0 is possible. Or "installAfter": "jdk" in case we have install on JBoss EAP 6.3.0